.pipelines/onebranch/pipeline.logging.geneva.bootstrapper.pullrequest.yml

```diff
@@ -0,0 +1,46 @@
+#################################################################################
+#                               OneBranch Pipelines                             #
+# This pipeline was created by EasyStart from a sample located at:              #
+#   https://aka.ms/obpipelines/easystart/samples                                #
+# Documentation:  https://aka.ms/obpipelines                                    #
+# Yaml Schema:    https://aka.ms/obpipelines/yaml/schema                        #
+# Retail Tasks:   https://aka.ms/obpipelines/tasks                              #
+# Support:        https://aka.ms/onebranchsup                                   #
+#################################################################################
+
+trigger: none
+pr: none
+
+variables:
+  Cdp_Definition_Build_Count: $[counter('', 0)] # needed for onebranch.pipeline.version task https://aka.ms/obpipelines/versioning
+  LinuxContainerImage: centos:centos7   # Docker image which is used to build the project https://aka.ms/obpipelines/containers
```

Comment:
bye bye FIPS...

Reply:
Note: there are pipelines which already got merged wtih this image.

Reply:
I didn't realize there was a dedicated image for this - I can change the image for these two pipelines and then create another PR to fix the pipelines that were already merged.

Reply:
Images have been updated - going to run the pipelines again just to make sure.

Reply:
For clarity - I changed these to use the "cdpxlinux.azurecr.io/global/ubuntu-1804-all:5.0" image. Is that ok or is there a separate image that I should be using?

I was informed that the "cdpxlinux.azurecr.io/global/ubuntu-1804-all:5.0" image was only used because it could build images while the default OneBranch pipeline centos image could not.